src/redux-state/extraReducers.ts

```diff
@@ -0,0 +1,14 @@
+import { ActionReducerMapBuilder } from "@reduxjs/toolkit"
+import { IslandState } from "redux-state"
+import { fetchPopulation, initRealmsDataFromContracts } from "./thunks/island"
+
+const extraReducers = (builder: ActionReducerMapBuilder<IslandState>) => {
```

Comment:
in the `extension` we are keeping extra reducers in the same redux slice file as the slice it operates on. I think it would make sense to do it here as well because we may save some time debugging things later if we will see full picture in one file.